content/_help/get-started/create-your-account._en.md

```diff
@@ -42,12 +42,12 @@ Follow these steps to create your Login.gov account.
    * Phone call
    * Backup codes
 
-   [Learn more about each authentication option](/help/get-started/authentication-options/) to choose the one that is right for you.
+   [Learn more about each authentication option]({{ site.baseurl }}/help/get-started/authentication-options/) to choose the one that is right for you.
 7. Success! Once you have authenticated, you have created your Login.gov account.
 
 You will be taken to your Login.gov account page or the government agency you are trying to access. You can add additional authentication methods, email addresses or update your information on the Login.gov account page.
 
 ## Related articles
 
-* [Authentication options](/help/get-started/authentication-options/)
-* [Manage your account](/help/manage-your-account/overview/)
\ No newline at end of file
+* [Authentication options]({{ site.baseurl }}/help/get-started/authentication-options/)
```

Comment:
is this going to be difficult to maintain using NetlifyCMS?

Reply:
We need to communicate this to the folks maintaining the content. 
In order for internal links to work in our preview environment, we need to preface our relative url's with the jekyll liquid baseurl variable.

Reply:
Have we tried adding a [`<base>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base) to the HTML for preview builds? Then we could let these stay the same and simplify content editing

Reply:
I'll test that out. 
There might be some weird issues with jekyll-redirect-from. In order to set a base, as best as I understand, I need a full url, not just the subdirectory. So it'd have to be something like `<base href="{{ site.url | append "/" | append {{ base.url }}>`. I'm not sure if federalist will populate the site.url correctly.

Reply:
It seems like `<base>` would only apply to relative links, not root-relative paths (e.g. `/foo/bar`).

Maybe we could use a combination of tricks we've used elsewhere (https://github.com/18F/identity-style-guide/pull/274, https://github.com/18F/identity-style-guide/pull/252) to inject the Federalist base URL to Markdown links dynamically with a plugin, so they could be written as root-relative, but prepended in Federalist previews?

```rb
module Kramdown
  module Parser
    class Kramdown
      prepend(Module.new do
        def add_link(el, *args)
          el.attr['href'] = ENV['BASEURL'] + el.attr['href'] if ENV['BASEURL'] && el.attr['href']&.start_with?('/')
          super(el, *args)
        end
      end)
    end
  end
end
```